<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvestigationItem</class>
 <widget class="QWidget" name="InvestigationItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="nameButton">
         <property name="toolTip">
          <string>Click to modify the box name</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addMemoButton">
         <property name="toolTip">
          <string>Click to add memo to this box</string>
         </property>
         <property name="text">
          <string>Add memo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hideButton">
         <property name="toolTip">
          <string>Click to hide this box</string>
         </property>
         <property name="text">
          <string>Hide</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="toolTip">
          <string>Click to delete this box</string>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveToNewButton">
         <property name="toolTip">
          <string>Click to move this box and its subtree to a new investigation</string>
         </property>
         <property name="text">
          <string>Move to new</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newButton">
         <property name="toolTip">
          <string>Click to move this box to a new investigation as the start node</string>
         </property>
         <property name="text">
          <string>New investigation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="runQueryLabel">
         <property name="text">
          <string>Run query from</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="positionLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="queryEdit">
         <property name="toolTip">
          <string>Type semantic query here</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="runQueryButton">
         <property name="toolTip">
          <string>Click to run the semantic query from the given position</string>
         </property>
         <property name="text">
          <string>Run query</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="predefLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Predefined queries</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="predefComboBox">
         <property name="toolTip">
          <string>List of predefined queries</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
